hw/ip/otbn/doc/_index.md

```diff
@@ -557,6 +557,15 @@ This is to allow OTBN applications to store sensitive information in the other h
 Each memory write through the register interface updates a checksum.
 See the [Memory Load Integrity]({{< relref "#mem-load-integrity" >}}) section for more details.
 
+### Instruction Prefetch
+
+OTBN employs an instruction prefetch stage which allows register file inputs to be taken directly from registers by pre-decoding instructions in the prefetch stage.
+This is required to enable the blanking SCA hardening measure (NOT YET IMPLEMENTED OR DOCUMENTED).
+Its operation is entirely transparent to software.
+It does not speculate and will only prefetch where the next instruction address can be known.
+This results in a stall cycle for all conditional branches and jumps as the result is not predicted nor known ahead of time.
+Instruction data held in the prefetch buffer is unscrambled but uses the integrity protection described in [Data Integrity Protection]({{<relref "#design-details-data-integrity-protection">}}).
```

Comment:
Nit: I'd cheat and say "Instruction bits" to side-step questions about whether data is singular or a mass noun.